charts/capsule/templates/mutatingwebhookconfiguration.yaml

```diff
@@ -32,6 +32,7 @@ webhooks:
       - v1
       operations:
       - CREATE
+      - UPDATE
```

Comment:
Changes on Chart requires a version bump up in order to build a new release.

Reply:
should i also update `appVersion` in chart?

Reply:
Since we don't have a new image, nor a release candidate, we can skip the chart bump.